### ISSUE TITLE
Confirm MEMS model configuration

### ISSUE THREAD

**Issue by kdorheim** [Microbial-Explicit-Model/MEMC#21]
Right now the the implementation of MEMS in our package uses the following configuration.

|model |DOMuptake |POMdecomp |MBdecay |
|:-----|:---------|:---------|:-------|
|MEMS  |LM        |LM        |LM      |

1) We need to confirm that this the correct combination of the dynamics.
2) Do we have the correct citation? 

Robertson, A. D., Paustian, K., Ogle, S., Wallenstein, M. D., Lugato, E., and Cotrufo, M. F.: Unifying soil organic matter formation and persistence frameworks: the MEMS model, Biogeosciences, 16, 1225–1248, 2019.

**Comment by jianqiuz** [comment]
Yes for MEMS 1.0. But it doesn't have a microbial pool. Only MEMS 2.0 included a microbial pool. We don't need to include this model for now

**Comment by kdorheim** [comment]
Would MEMS 2.0 have the microbial pool, and would it make sense to upgrade the memc model configuration to MEMS2?
